### PR TITLE
Updated 40x-dv hash to include rv32e assert file

### PR DESCRIPTION
**Pull request by silabs-mateilga**

### CODE REVIEW

File: bin/clonetb
Comment (silabs-robin) [inline bin/clonetb:1]: ci_check passes and formal still runs?

Comment (silabs-mateilga) [reply bin/clonetb:1]: It did when we merged the PR to 40x-dv, I can rerun the checks now, if necessary?

Comment (silabs-robin) [reply bin/clonetb:1]: Oh ok. If it was ran on cv32e40x/dev then its the same.

**Review by silabs-robin — APPROVED** [review]
Approving based on:
Nice to get the hash updated.
ci_check and formal was good on https://github.com/openhwgroup/cv32e40x-dv/pull/35 .

**Pull request merged**